particle/particle/particle.py

```diff
@@ -452,13 +452,13 @@ def from_search_list(cls, filter_fn=None, particle=None, **search_terms):
         Search for a particle, returning a list of candidates.
 
         The first and only positional argument is given each particle
-        candidate, and returns true/false. Example:
+        candidate, and returns True/False. Example:
 
             >>> Particle.from_search_list(lambda p: 'p' in p.name)
             # Returns list of all particles with p somewhere in name
 
-        You can also pass particle=True/False to force a particle or antiparticle. If
-        this is not callable, it will do a "fuzzy" search on the name. So this is identical:
+        You can pass particle=True/False to force a particle or antiparticle.
+        IF this is not callable, it will do a "fuzzy" search on the name. So this is identical:
```

Comment:
Why IF instead of If?

Reply:
Good spot - Atom played my tricks :S. Of course it should be "If" ... will fix ...

Reply:
Done.